tests/test_install.py

```diff
@@ -27,10 +28,28 @@ def setUp(self):
     def tearDown(self):
         self.get_dirs_patch.stop()
 
+    def _assert_direct_url(self, directory, package, version, expected_editable):
+        direct_url_file = (
+            self.tmpdir
+            / 'site-packages'
+            / '{}-{}.dist-info'.format(package, version)
+            / 'direct_url.json'
+        )
+        assert_isfile(direct_url_file)
+        with direct_url_file.open() as f:
+            direct_url = json.load(f)
+            assert direct_url.get('url').startswith('file:///')
+            assert direct_url.get('url') == directory.as_uri()
+            assert 'dir_info' in direct_url
+            assert direct_url.get('dir_info').get('editable') is expected_editable
```

Comment:
```suggestion
            assert direct_ur['url'].startswith('file:///')
            assert direct_url['url'] == directory.as_uri()
            assert 'dir_info' in direct_url
            assert direct_url['dir_info']['editable'] is expected_editable
```

IMO, if the key is ever missing, a KeyError makes that clear better than getting `None` and trying to check it.

Reply:
done